app/routes/orderRoutes.js

```diff
@@ -5,9 +5,12 @@ const asyncHandler = require('../utils/asyncHandler');
 const router = express.Router();
 
 // View all orders of a strategy
-router.get('/orders', asyncHandler(OrderController.getAllOrders));
+router.get('/api/v1/orders', asyncHandler(OrderController.getAllOrders));
 
 // Buy a new order for a strategy
-router.post('/orders', OrderController.buyNewOrder);
+router.post('/api/v1/orders', OrderController.buyOrder);
```

Comment:
Why there's no validators?

Please check other interfaces as well.

Reply:
place and sell were both automatic scheduled tasks. Making them as interfaces without validators is just for testing now.